data/dashboards/data.yaml

```diff
@@ -51,7 +51,7 @@ items:
           - label: Terms added to PCL
             count: 461
   - type: GridContainer
```

Comment:
You can remove the grid container since it has only 1 column. Just move the vega containers to the same level as the image container